polkadot/runtime/parachains/src/builder.rs

```diff
@@ -702,11 +704,8 @@ impl<T: paras_inherent::Config> BenchBuilder<T> {
 			.map(|i| {
 				let AssignmentProviderConfig { ttl, .. } =
 					scheduler::Pallet::<T>::assignment_provider_config(CoreIndex(i));
-				// Load an assignment into provider so that one is present to pop
-				let assignment = <T as scheduler::Config>::AssignmentProvider::get_mock_assignment(
-					CoreIndex(i),
-					ParaId::from(i),
-				);
+				let assignment =
```

Comment:
Why was this changed? And how is it supposed to work? We don't know the underlying assigner, hence we can't know whether the pop_assignment_for_core will yield any assignment. Depending on configuration this unwrap will fail.

Reply:
This is from a Git mishap. I'll clean it up.